api/envoy/config/transport_socket/raw_buffer/v2/raw_buffer.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package envoy.config.transport_socket.raw_buffer.v2;
+
+option java_package = "io.envoyproxy.envoy.config.transport_socket.raw_buffer.v2";
+option java_outer_classname = "RawBufferProto";
+option java_multiple_files = true;
```

Comment:
You probably need to `git add` the new v3alpha protos in `api/` and `generated_api_shadow/`. Also, please make sure that the migration annotations are included to prevent regression here, see #9580  (I think I'll add these checks today, since this requirement is non-intuitive).

Reply:
Done.